### DISCUSSION TITLE
engage / disengage motor

### DISCUSSION THREAD

**Discussion by gregorysemah** [luni64/TeensyStep#156] [General]
Hello @luni64 
User of AccelStepper, i wanted to test TeensyStep as a potential replacement for it.
After reading the docs, i was wondering if it would be possible  to disengage/engage motor (as disableoutput/enableoutput methods from accelStepper).
Maybe set to HIGH the enablePin from the driver is enough? no control on that from your lib?
If this question has already been asked, sorry for that.. :)

Thanks again for your answer

**Comment by luni64** [comment]
Enabling/disabling is really just setting the corresponding pin high/low. Since polarity / availability totaly depends on the used driver I decided that supporting this in the lib is not worth it.
